tests/unit/cag/test__utils.py

```diff
@@ -6,7 +6,17 @@
 import pytest
 
 from sdv.cag._errors import PatternNotMetError
-from sdv.cag._utils import _validate_table_and_column_names
+from sdv.cag._utils import (
+    _is_list_of_strings,
+    _remove_columns_from_metadata,
+    _validate_table_and_column_names,
+    _validate_table_name_if_defined,
+)
+from sdv.metadata.metadata import Metadata
+
+expected_not_single_table = re.escape(
```

Comment:
Why move this out of the method?

Reply:
This was an accident. Fixed:
https://github.com/sdv-dev/SDV/pull/2409/commits/ab7486461cdacaa4a32ab55016ba3204f780c406#diff-4c052b8f00ae5eb42d72e0fe171cf5ad9e1901e9739535a0dbe7c25b0070fc39R27